src/Pages/HomePage/Homepage.js

```diff
@@ -0,0 +1,170 @@
+import './HomePage.css';
+import Navbar from '../../Components/Navbar/Navbar';
+import Footer from '../../Components/Footer/Footer';
+import ActivitiesNorthLebanon from '../../Components/ActivitiesNorthLebanon/ActivitiesNorthLebanon';
+import Search from '../../Components/Search/Search';
+import NeedATourGuide from '../../Components/NeedATourGuide/NeedATourGuide';
+import CityCard from '../../Components/Card/CityCard/CityCard';
+import JoinOurTeam from '../../Components/Card/JoinOurTeam/JoinOurTeam.js';
+import ActivityCard from '../../Components/Card/ActivityCard/ActivityCard';
+import { Suspense } from 'react';
+import { Carousel } from 'antd';
+import { LeftCircleOutlined, RightCircleOutlined } from '@ant-design/icons';
+
+const HomePage = () => {
+  const guides = [
```

Comment:
can you move this to a folder in the `src` name it `mockdata` and inside it add file called `data.js` and then export this array from there so we can keep it clean here